### DISCUSSION TITLE
how to use "flux1-canny-dev-lora" with diffusers

### DISCUSSION THREAD

**Discussion by todochenxi** [huggingface/diffusers#10149] [General]
I got a error: laod error:: Invalid LoRA checkpoint.

**Comment by anandguptacv** [comment]
# Loading LoRA Weights in Your Pipeline
When implementing LoRA (Low-Rank Adaptation) in your model, ensure that you have the correct path to your LoRA weights. Here’s a guideline on how to load them properly:

Check Your LoRA Path: Verify that the path specified for the LoRA weights is correct.

Loading Weights: Use the following code snippet to load your LoRA weights into the pipeline:

### Load LoRA weights
pipe.load_lora_weights(path_of_lora, adapter_name="default")
Using the Adapter: Make sure to utilize the adapter corresponding to your specific LoRA configuration. If your implementation has a different adapter name, replace "default" with your specific adapter name.
`
Example
Here’s a complete example:

### Define the path to your LoRA weights
path_of_lora = "path/to/your/lora/weights"

#### Load the weights into the pipeline
pipe.load_lora_weights(path_of_lora, adapter_name="your_adapter_name")

`

**Comment by a-r-r-o-w** [comment]
@todochenxi This should now be supported as #9999 is merged. Could you try again?

**Reply by todochenxi** [reply]
After upgrading PEFT and reinstalling Diffusers, everything is now working properly. Thank you very much for your help.
